src/main/java/core/basesyntax/service/action/BalanceAction.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.action;
+
+import core.basesyntax.model.Account;
+
+public class BalanceAction implements ActionHandler {
+    private Account account;
```

Comment:
The `Account` object is declared but not initialized. This will lead to a `NullPointerException` when calling `setBananaBalance` or `setAppleBalance`. Consider initializing the `Account` object, either in the constructor or through dependency injection.